gulpfile.js

```diff
@@ -151,14 +157,36 @@ gulp.task('exportPatterns', ['patternsExport:clean'], () => {
 
 });
 
+gulp.task('sharedConfig:clean', () => {
+  return del(config.files.src.derivedConfigs);
+});
+
+gulp.task('distributeSharedConfig', ['sharedConfig:clean'], (done) => {
+  exec('node ./libero-config/bin/distributeConfig.js', (err, stdout, stderr) => {
```

Comment:
Can a sub-process be avoided?

Reply:
Possibly. I originally tried to integrate it into the gulp process more effectively, but it didn't work. I think it's time for another go though.